README.md

```diff
@@ -27,26 +27,66 @@ export LIBCLANG_PATH=/usr/lib/clang/4.0/lib
+   > Note: LLVM under 14 is not able to build mozjs-sys.
 
-   ```shell
-   "c:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvars64.bat"
+3. Download and install `Visual Studio 2022` with `C++ desktop development` with following features:
+   - Windows 11 SDK
+   - ATL
+   - MFC
+  
+    If you want to build in command line, you can download [vs_buildtools.exe](https://aka.ms/vs/17/release/vs_buildtools.exe) linked from and run the following command:
+   ```
+   vs_BuildTools.exe^
+      --add Microsoft.Component.MSBuild^
+      --add Microsoft.VisualStudio.Component.CoreBuildTools^
+      --add Microsoft.VisualStudio.Workload.MSBuildTools^
+      --add Microsoft.VisualStudio.Component.Windows11SDK^
+      --add Microsoft.VisualStudio.Component.VC.CoreBuildTools^
+      --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64^
+      --add Microsoft.VisualStudio.Component.VC.Redist.14.Latest^
+      --add Microsoft.VisualStudio.Component.VC.ATL^
+      --add Microsoft.VisualStudio.Component.VC.ATLMFC^
+      --add Microsoft.VisualStudio.Component.VC.CoreIde^
+      --add Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core^
+      --add Microsoft.VisualStudio.Workload.VCTools
    ```
 
-4. Set the following environment variables according to where you installed
-   the dependencies above:
+1. Install Python3 and add into PATH.
 
+2. Set the following environment variables according to where you installed
+   the dependencies above:
    ```shell
-    set LIBCLANG_PATH=C:\Program Files\LLVM\lib
-    set MOZTOOLS_PATH=C:\path\to\moztools-4.0
+    $env:LIBCLANG_PATH="C:\Program Files\LLVM\lib"
+    $env:MOZTOOLS_PATH="C:\path\to\moztools-4.0"
+    $env:CC="clang-cl"
+    $env:CXX="clang-cl"
+    $env:LD="lld-link"
+   ```
+   for better experience, you can set LLVM and MozTools in your system PATH and ENV variables.
+
+3. If cargo build throw out error shows command in `maybe-configure` failed,
+   ```
+   error: failed to run custom build command for `mozjs-sys v0.1.0 ...
+   // bunch of error messages
+   maybe-configure:
+	[[ $(JSSRC)/configure -ot $(JSSRC)/configure.in ]] && touch $(JSSRC)/configure || true
+	[[ $(JSSRC)/old-configure -ot $(JSSRC)/old-configure.in ]] && touch $(JSSRC)/old-configure || true
+	! [[ $(JSSRC)/configure.in -ot $(JSSRC)/configure ]] && touch $(JSSRC)/configure || true
+	! [[ $(JSSRC)/old-configure.in -ot $(JSSRC)/old-configure ]] && touch $(JSSRC)/old-configure || true
+	if [[ $(JSSRC)/configure -nt config.status ]] ; then \
+	  CC="$(CC)" CFLAGS="$(CFLAGS)" \
+	  CPP="$(CPP)" CPPFLAGS="$(CPPFLAGS)" \
+	  CXX="$(CXX)" CXXFLAGS="$(CXXFLAGS)" \
+	  AS="$(AS)" AR="$(AR)" \
+	  $(JSSRC)/configure $(strip $(CONFIGURE_FLAGS)) || (cat config.log && exit 1) ; \
+	fi
+
+   exit code: not 0
    ```
+   try to modify `mozjs/mozjs/js/src` replace line  `PYTHON3="${PYTHON3:-python3}"` with `PYTHON3="${PYTHON3:-python}"`
```

Comment:
I think by setting the `PYTHON3` environment variable above you can remove this workaround.

Reply:
sure! shall I remove this?
but btw I am not sure env is effected in this build file

Reply:
Does properly setting the `PYTHON3` environment variable remove the need for manually editing the build files? If that's the case, I think we should definitely remove this.